### ISSUE TITLE
Switching `master` to `main` closed some PRs

### ISSUE THREAD

**Issue by Falke-Design** [Leaflet/Leaflet#7923]
PRs which are auto-closed:
- #7567 
- #3955 
- #6066 
- #6248 
- #7364 
- #6676 
- #6935

**Comment by jonkoops** [comment]
There are all the ones I am seeing in my notifications.

**Comment by Falke-Design** [comment]
Yes same for me. But per E-Mail I get some more closes but I can't open them:

![grafik](https://user-images.githubusercontent.com/19800037/150134773-8f9f0559-a040-47ef-b62d-e07673259840.png)
https://github.com/Leaflet/Leaflet/pull/7671
